getting-started/installation/adding-private-nuget-feed.md

```diff
@@ -44,48 +59,22 @@ If you work with Visual Studio Code on Linux or Mac OS, use the Nuget CLI to set
 
 1. Download the latest [NuGet executable](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe).
 1. Open a Command Prompt and change the path to the `nuget.exe` location.
-1. The command from the example below stores a token in the `%AppData%\NuGet\NuGet.config` file. Your original credentials cannot be obtained from this token.
-
-	````powershell
-NuGet Sources Add -Name "telerik.com" -Source "https://nuget.telerik.com/v3/index.json" ^
-	-UserName "your login email" -Password "your password"
-````
-
-	If you are unable to connect to the feed by using encrypted credentials, store your credentials in clear text.
+1. The command from the example below stores your API key in the `%AppData%\NuGet\NuGet.config` file.
 
-	````powershell
+````powershell
```

Comment:
Start ```` with a tab as in the original snippet as it is in OL - see the Wiki for details.

Reply:
I'm not finding anything about the required tab. Could this requirement be missing?